Plugins/IntelliJ/gradle.properties

```diff
@@ -3,9 +3,9 @@
 
 pluginGroup = com.shopify.testify
 pluginName = Android Testify - Screenshot Instrumentation Tests
-pluginVersion = 0.1.2
 pluginSinceBuild = 193
 pluginUntilBuild = 211.*
+javaVersion = 1.8
```

Comment:
Do we actually have to build the plugin with Java 8? Everything else is on 11 now.

Reply:
I added this in response to a build error. I suspect that, yes, it requires 8.
There's a bunch of places that explicitly request java 8. 🤷 

```
./Plugins/IntelliJ/build.gradle.kts:59: sourceCompatibility = "1.8"
./Plugins/IntelliJ/build.gradle.kts:60: targetCompatibility = "1.8"
./Plugins/IntelliJ/build.gradle.kts:64: kotlinOptions.jvmTarget = "1.8"
```